.github/workflows/test.yml

```diff
@@ -27,4 +27,4 @@ jobs:
       - name: Add wasm32 target
         run: rustup target add wasm32-unknown-unknown
       - name: Test
-        run: cargo test --all
+        run: cargo test --all --features unstable
```

Comment:
Let's add a separate step for this or add an `&&`, to make sure we don't break code *without* the feature. 

And, while we are at it, `s/all/workspace`, `--all` is deprecated.

Reply:
Yeah, so the reason I didn't was that the `unstable` feature is purely additive and the CI is already very time-consuming. @mikedotexe have any thoughts here?

Reply:
>Yeah, so the reason I didn't was that the unstable feature is purely additive

This is exactly the thing which is good to verify on CI. But I agree that this is not critical. 

> CI is already very time-consuming. 

Oh wow, 20min for CI for the project of this scale does seem outrageous. rust-analyzer's CI takes 7ish minutes ([example](https://github.com/rust-analyzer/rust-analyzer/runs/2620142057)), so there's probably a room for improvement here

Reply:
Yeah my gut is to have a separate step as well, although CI that would add even more time. I know with Travis we could add automated weekly/monthly checks, perhaps we could do this and have our team get alerts. Thoughts, @austinabell ?